metrics/setup.go

```diff
@@ -0,0 +1,67 @@
+package metrics
+
+import (
+	"net/http"
+	"time"
+
+	ma "gx/ipfs/QmNTCey11oxhb1AxDnQBRHtdhap6Ctud872NjAYPYYXPuc/go-multiaddr"
+	manet "gx/ipfs/QmZcLBXKaFe8ND5YHPkJRAwmhJGrVsi1JqDZNyJ4nRK5Mj/go-multiaddr-net"
+
+	prom "github.com/prometheus/client_golang/prometheus"
+	"go.opencensus.io/exporter/prometheus"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/zpages"
```

Comment:
These also need to be gxed.

Reply:
I will gx all opencensus and prometheus imports